apps/btc_family/btc_helpers.h

```diff
@@ -29,6 +29,10 @@
 #define BTC_ACC_XPUB_DEPTH 3
 #define BTC_ACC_ADDR_DEPTH 5
 
+#define SATOSHI_PER_BTC 100000000
+
+#define OP_RETURN 0x6A
```

Comment:
Some explanation for this macro would be better

Reply:
yes. will add a reference.

Reply:
added